blackberggroup/VA-OAM
# Feature/rb 6627643780/dynamic newsletter article presentation

## What type of PR is this? (check all applicable)


## Description

- Initial setup of dynamic newsletter functionality. Previous articles were hardcoded in the project and now we are pulling them in via JSON from SharePoint. 

## Related Tickets & Documents
- Monday Task #6627643780

## QA Instructions, Screenshots, Recordings

-View the main News and Events page to verify articles are populating 
-View a news detail page and verify data is populating

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _includes/news-and-events/newsletter-articles-dynamic.html
 - _site/article/index.html
 - _site/assets/js/colcade.js
 - _site/assets/js/min-masonry.js
 - _site/assets/js/mini-masonry.js
 - _site/assets/js/news-article.js
 - _site/assets/js/news-functions.js
 - _site/assets/js/news-json-render-dynamic.js
 - _site/news-and-events/article/index.html
 - article/index.html
 - assets/js/colcade.js
 - assets/js/news-article.js
 - assets/js/news-functions.js
 - assets/js/news-json-render-dynamic.js
 - news-and-events/article/index.html
 - _includes/news-and-events/articles.html
 - _site/assets/css/styles.css
 - _site/assets/sass/vha/components/card/_index.scss
 - _site/assets/sass/vha/pages/news/_index.scss
 - _site/news-and-events/index.html
 - _site/news-and-events/podcasts/How-3D-Printing-is-Transforming-Health-Care/index.html
 - assets/css/styles.css
 - assets/img/.DS_Store
 - assets/sass/vha/components/card/_index.scss
 - assets/sass/vha/pages/news/_index.scss
 - news-and-events/index.html
